packages/scoutgame/src/builders/getTodaysHotBuilders.ts

```
@@ -111,7 +118,9 @@ export async function getTodaysHotBuilders(): Promise<BuilderInfo[]> {
       rank: builder.userWeeklyStats[0]?.rank || -1,
       last7DaysGems: ((builder.builderCardActivities[0]?.last7Days as unknown as Last7DaysGems) || [])
         .map((gem) => gem.gemsCount)
-        .slice(-7)
+        .slice(-7),
+      contractAddress: builder.builderNfts[0]?.contractAddress || '',
```

instead of passing an empty string to circumvent Typescript logic, contractAddress should probably be optional or u should throw an error

---

Dropped contract address as it was part of iterations